src/resources/nfts/simplehash/utils.ts

```diff
@@ -394,7 +398,7 @@ export function simpleHashNFTToInternalNFT(nft: ValidatedSimpleHashNFT): NFT {
           f.marketplace_id
         )!,
         paymentToken: {
-          address: f.payment_token.address,
+          address: f.payment_token.address ?? ETH_ADDRESS,
```

Comment:
bc I changed type from `NFTPaymentToken` to `Asset`. the only difference between these two types is that eth address is represented by `ETH_ADDRESS` instead of `null`

Reply:
Do we really want to default to a default addy if none is provided?

Reply:
in this case it's not exactly defaulting– a SimpleHash payment token `address` is of type `string | null`, so `null` is actually intentionally used to represent the "eth address" (which doesn't exist since it's not an erc20)